docroot/themes/humsci/humsci_basic/tests/sass-specs/tools/_functions.animation.scss

```diff
@@ -0,0 +1,90 @@
+@include describe('HB Animation Function') {
+
+  // Colorful
+  @include describe('the hb-transition() function used with Colorful') {
+    $hb-current-theme: 'colorful' !global;
+
+    @include it('returns the theme variable values for the transition-duration and transition-timing-function properties for the Colorful theme') {
+      @include assert {
+        @include output {
+          @include hb-colorful {
+            transition: hb-transition(color);
+          }
+        }
+        @include contains {
+          transition: color 150ms ease-in-out;
+        }
+      }
+
+      @include it('does not return styles because the mixin does not match the $hb-current-theme') {
```

Comment:
I dont know that you need this test. You are trying to test the hb-transition function. Maybe if you demonstrated that that function didn't return anything here you could do a test about that? but i guess it would always return something because we have the theme set by default